test/libsolidity/util/TestFileParser.cpp

```diff
@@ -309,7 +346,7 @@ Parameter TestFileParser::parseParameter()
 	if (accept(Token::Boolean))
 	{
 		if (isSigned)
-			throw TestParserError("Invalid boolean literal.");
+			BOOST_THROW_EXCEPTION(TestParserError("Invalid boolean literal."));
```

Comment:
I think @chriseth suggested to replace all throws, but perhaps you want to create a simple PR which replaces all throws and all asserts? We can merge that already.

Reply:
Could also include the boost::bind->std::bind change (https://github.com/ethereum/solidity/pull/10867#discussion_r578690040) in that PR.

Reply:
Ah ok, I just replace the throws in `TestFileParser.cpp`. @chriseth did you mean to replace all throws used?

Reply:
However, I added a commit that replaces throws.

Reply:
I just meant to have a new PR which replaces the throws/asserts/bind in TestFile*, and we can merge it right away.

Reply:
Sure, I could create another PR for this. I think we could always use `BOOST_THROW_EXCEPTION` instead of using `throw`.